common/test/common_tests/colors_test.cljc

```diff
@@ -53,7 +53,13 @@
 
 #?(:cljs
    (t/deftest format-hsla
-     (t/is (= "210, 50%, 1%, 1" (colors/format-hsla [210.0 0.5 0.00784313725490196 1])))))
+     (t/is (= "210, 50%, 0.78%, 1" (colors/format-hsla [210.0 0.5 0.00784313725490196 1])))
+     (t/is (= "220, 5%, 30%, 0.8" (colors/format-hsla [220.0 0.05 0.3 0.8])))))
+
+#?(:cljs
```

Comment:
So making the format-number available for CLJ, that tests can also be excluded from the conditional